stage1/init/common/pod.go

```diff
@@ -14,11 +14,10 @@
 
 //+build linux
 
-package main
+package stage1initcommon
```

Comment:
I don't like this kind of names, should be just "common" (or better, a meaningful name): https://golang.org/doc/effective_go.html#names